frontend/src/routes/(authenticated)/devices/+page.svelte

```diff
@@ -141,7 +143,8 @@
 						saveState();
 					}}
 				/>
-				<TableBodyEditCell bind:value={device.data.targetHost} onEnter={() => saveState()} />
+				<TableBodyCell>{device.target ? device.target : $t('devices.unknown-target')}</TableBodyCell
+				>
```

Comment:
we still want to edit the target host manually. Editing should update the database

Reply:
We need a general rework here. I guess @Florian-S-Dev is currently working on separating the device overview table into sub pages for each device